examples/openapi-tools/quickstart-se/README.md

```diff
@@ -0,0 +1,383 @@
+# Helidon OpenAPI Generator example for Helidon SE server and client
+
+The goal of this example is to show how a user can easily create a Helidon SE server or client from an OpenAPI document using OpenAPI Generator.  
+
+Here we will show the steps that a user has to do to create Helidon SE server and client using OpenAPI Generator and what has to be done to make the generated server and client fully functional.
+
+For generation of our projects we will use `openapi-generator-cli.jar` that can be downloaded form the maven repository (instructions and other options can be found [here](https://openapi-generator.tech/docs/installation)) and OpenAPI document `quickstart.yaml` that can be found next to this `README.md`.
+
+## Build, prepare and run the Helidon SE server
+
+To generate Helidon SE server at first we create `se-server` folder and then inside it we run the following command where `path-to-generator` is the directory where you downloaded the generator CLI JAR file and `path-to-openapi-doc` is the folder where `quickstart.yaml` is located:
+```bash
+java -jar path-to-generator/openapi-generator-cli.jar \
+          generate \
+          -g java-helidon-server \  
+          --library se \
+          -i path-to-openapi-doc/quickstart.yaml
+```
+
+When this command finishes its work in the folder `se-server` we will find the generated project where the most interesting parts are located inside `api` and `model` packages.
+The package `api` contains interfaces that represent endpoints for our server and implementations with stubs for them. 
+These implementations we need to change to implement our business logic.
+The package `model` contains classes that represent transport objects that will be used by our endpoints to receive requests and send responses.
+
+Let's change a little class `MessageServiceImpl` for our example :
+1) Add field that will contain default message for our endpoints :
+```java
+    private final AtomicReference<Message> defaultMessage = new AtomicReference<>();
+```
+2) Add default constructor to the class :
+```java
+    public MessageServiceImpl() {
+        Message message = new Message();
+        message.setMessage("World");
+        message.setGreeting("Hello");
+        defaultMessage.set(message);
+    }
+```
+4) Replace implementation of the method `public void getDefaultMessage(ServerRequest request, ServerResponse response)` by this:
+```java
+        response.send(defaultMessage.get());
+```
+5) Replace implementation of the method `public void getMessage(ServerRequest request, ServerResponse response)` by this:
+```java
+        String name = request.path().param("name");
+        defaultMessage.get().setMessage(name);
```

Comment:
In my  comment in my previous review on the `quickstart-mp/README.md` I mentioned that _both_ servers had the same problem; they both changed the default greeting's name in the `getMessage` method which should not happen. 

I probably should have repeated that comment here at that time.

The latest changes to the PR fix the MP example but not the SE example here. This SE `getMessage` code still changes the default name.

Reply:
Sorry, I forgot to update README.md